doc/probe.txt

```diff
@@ -186,6 +186,13 @@ OPTIONS                                                        *probe-options*
 
     g:probe_ignore_files still functions normally when this option is set.
 
+|g:probe_use_gitignore|                              *g:probe_use_gitignore*
+    Use the current repositories .gitignore file to when scanning for files.
```

Comment:
More accurately we're using git's "ignore settings", but I don't know what the right name for them is. The global and `info/exclude` files are taken into account, too, right?